src/bia_bob/_utilities.py

```diff
@@ -90,12 +94,13 @@ def create_system_prompt():
     
     ## Explanations and code
     
-    Before writing the code, provide a concise step-by-step plan 
-    of what the code will be going to do. Always provide this text explanation first.
+    Before writing any code in any language, provide a concise step-by-step plan 
+    of what the code will be going to do. Always provide the plan first.
     This plan must not contain any "`" characters and should be written in plain text.
     Then print the code.
+    There must be only one code block.
     Importantly, the code block must start with the line: 
-    ```python
+    ```code
```

Comment:
This is kind-of-invalid. As Markdown defines `python` in this context, but no `code`. You find a list [here](https://www.rubycoloredglasses.com/2013/04/languages-supported-by-github-flavored-markdown/).
```suggestion
    ```
```